src/lib/chips/chip-list.ts

```diff
@@ -655,6 +653,7 @@ export class MatChipList extends _MatChipListMixinBase implements MatFormFieldCo
 
   /** When blurred, mark the field as touched when focus moved outside the chip list. */
   _blur() {
+    this._keyManager.setActiveItem(-1);
```

Comment:
Unit test?

Reply:
Added unit tests and fixed chip emits multiple `_onFocus` events